### ISSUE TITLE
The project should invite the architect, not the other way around

### ISSUE THREAD

**Issue by amihaiemil** [zerocracy/farm#573]
Currently, the client submits a [RFP](https://www.0crat.com/rfp) and waits for the best architect to respond. 

I think the client should interview them instead. ARCH is the most powerful role in a project, so the person has to really have the knowledge and experience for it. On the other hand, the reputation points are not a good metric for that: 2048 reputation is achievable in many ways, maybe I am a brilliant tester and built that only from reporting bugs. Same issue as the 50k StackOverflow reputations built on top of Jquery and CSS questions. They mean you're good, you know the model, you can be productive, but they certainly do not mean that you are a good software architect. 

So I, the client, pay a $64 non-refundable tax and I should just trust that reputation number. What if it turns out that the person lacks essential ARCH skills and cannot handle my project? What if no architect responds?

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#573](https://github.com/zerocracy/farm/issues/573) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://datum.zerocracy.com/pages/policy.html#29): +15 points just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), total is [+645](https://www.0crat.com/u/amihaiemil)

**Comment by yegor256** [comment]
@amihaiemil sounds reasonable. How about we ask the client to provide the skills he/she is looking for. Then, we match the RFP only with the architects who have those skills (see https://github.com/zerocracy/farm/issues/492). WDYT?

**Comment by amihaiemil** [comment]
@yegor256 yes, sounds ok. But then the client interviews them, he/she selects the architect and doesn't have to rely only on the reputation and/or those skills. Again, the fact that I contributed to 100 Java repos and 50 Kotlin ones does not mean that I have the skills to properly architect a software from all points of view.

**Comment by amihaiemil** [comment]
@yegor256 for instance, myself, I am confident in my OOP design skills + testing, I could architect a library or a small/medium-size application. But I kind of suck when it comes to databases. I can work on a DB access layer, but not sure I could design one to be fast, scalable etc etc. So someone may see my reputation and skills and figure I can deisgn some mission critical software for them -- we need to discuss and see what are the expectations and what I can offer realistically.

**Comment by yegor256** [comment]
@amihaiemil you will discuss, when you "buy" the RFP. So, first of all you will assess whether the project is doable for you. Then, you will pay for the RFP and email the client. Then, you will discuss with the client. They will check your resume, etc. Then, they will trust you the project. There have to be too many mistakes made, by all parties, in order for you (being "just a tester with a high reputation") to landed in the ARC position in a big enterprise project.

The only valid concern I see here is that the client may end up paying too many times for the RFP until they find the right guy. That's a valid concern. Maybe we should just lower the price to, say, $8 or $16. In that case they won't care too much and will just re-submit again and again.

**Comment by amihaiemil** [comment]
@yegor256 or charge the fee only after the project is settled, with an architect?

**Comment by yegor256** [comment]
@amihaiemil we don't know when it's "settled," maybe never. I think we should just charge smaller amount and that's it.

**Comment by 0crat** [comment]
The job [#573](https://github.com/zerocracy/farm/issues/573) is now out of scope

**Comment by yegor256** [comment]
@amihaiemil fixed: https://github.com/zerocracy/datum/commit/f16d6f704096c1c6a07ccf23279038f5838c009c
